hack/vf-netns-switcher.sh

```diff
@@ -0,0 +1,343 @@
+#!/bin/bash
+
+conf_file=""
+
+declare -a netnses
+
+declare -A pfs
+declare -A pcis
+declare -A pf_port_names
+declare -A pf_switch_ids
+
+TIMEOUT="${TIMEOUT:-2}"
+POLL_INTERVAL="1"
+
+while test $# -gt 0; do
+  case "$1" in
+
+   --netns | -n)
+      netnses+=("$2")
+      shift
+      shift
+      ;;
+
+   --pf | -d)
+      if [[ -z "${netnses[-1]}" ]];then
+          echo "ERROR: No netns is specified, please provide one using the --netns before using --pf."
+          exit 1
+      fi
+
+      pfs["${netnses[-1]}"]+="$2 "
+      shift
+      shift
+      ;;
+
+   --conf-file | -c)
+      conf_file=$2
```

Comment:
can you test if file exists and exit otherwise  ?

Reply:
done